packages/esm-ward-app/src/types/index.ts

```diff
@@ -141,7 +159,7 @@ export interface Encounter extends OpenmrsResourceStrict {
   voided?: boolean;
   visit?: Visit;
   encounterProviders?: Array<EncounterProvider>;
-  diagnoses?: any;
+  diagnoses?: Array<Diagnosis>;
```

Comment:
Is this diagnosis change tangential to the commit, or is this used somewhere within this commit, perhaps I'm missing it?